compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -1874,6 +1875,79 @@ struct DescendingSortX86DataSnippetByDataSize
       return a->getDataSize() > b->getDataSize();
       }
    };
+
+
+void OMR::X86::CodeGenerator::addItemsToRSSReport(uint8_t *oldColdCode, uint8_t *coldCode)
```

Comment:
Doesn't look like `oldColdCode` is used here.

Reply:
Right! Removed.